modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -1007,4 +1009,135 @@ test.describe('Manage objectFields through Objects Admin UI', () => {
 			)
 		).toBeVisible();
 	});
+
+	test('can edit external reference code of a custom field through the UI', async ({
```

Comment:
Can you unify the two tests in one? `can only edit external reference code of custom fields`, then you first check if system field ERC field is disabled (`expect(objectFieldsPage.externalReferenceCodeField).toBeDisabled()`) then you go to the custom field, edit it, save it and `expect(objectFieldsPage.externalReferenceCodeField).toHaveValue(ERCValue)`.

Reply:
Sure!